gulpfile.babel.js

```diff
@@ -121,7 +127,11 @@ gulp.task('dev', ['dev:doc', 'dev:static', 'dev:applystyles', 'dev:generate'], (
   gulp.watch('lib/styleguide.js', ['dev:generate']);
 });
 
-gulp.task('build', ['sass', 'js:app', 'js:vendor', 'html', 'assets']);
+gulp.task('buildDist', ['sass', 'js:app', 'js:vendor', 'html', 'assets']);
```

Comment:
Could you rename this task into `build:dist`. We use similar naming, with `:` in other tasks.